### PR TITLE
cmd: fix cmd TLS usage

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Fix `changefeed list` and `changefeed query` do not work when TLS is enabled.

### Check List 

Tests 

 - Integration test

Related changes

 - Need to cherry-pick to the release branch

### Release note

- Fix `changefeed list` and `changefeed query` do not work when TLS is enabled.

### CODE REVIEW

File: cmd/util.go [hunk 1]
```diff
@@ -133,13 +133,21 @@ func getOwnerCapture(ctx context.Context) (*capture, error) {
...
 
-func applyAdminChangefeed(ctx context.Context, job model.AdminJob) error {
+func applyAdminChangefeed(ctx context.Context, job model.AdminJob, credential *security.Credential) error {
 	owner, err := getOwnerCapture(ctx)
 	if err != nil {
 		return err
 	}
-	addr := fmt.Sprintf("http://%s/capture/owner/admin", owner.AdvertiseAddr)
-	resp, err := http.PostForm(addr, url.Values(map[string][]string{
+	scheme := "http"
```
Comment (liuzix) [inline cmd/util.go:141]: Need a new helper function for creating the Client?

Comment (overvenus) [reply cmd/util.go:141]: We have a helper function already, `httputil.NewClient`.

**Pull request merged**